cob_navigation_global/launch/2dnav_linear.launch

```diff
@@ -3,7 +3,7 @@
 
   <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
   <arg name="robot_env" default="$(optenv ROBOT_ENV !!NO_ROBOT_ENV_SET!!)"/>
-  <arg name="map" default="$(find cob_default_env_config)/$(arg robot_env)/map.yaml" />
+  <arg name="map" default="$(find cob_default_env_config)/envs/$(arg robot_env)/map.yaml" />
```

Comment:
In order to make those launch files more easily usable in the new structure, should we add a launch arg for the env package as well?